### PR TITLE
ignores empty rows

### PR DESCRIPTION
**Pull request by vmcvlad**

### CODE REVIEW

**Comment by aocneanu** [comment]
omg

**Comment by vmcvlad** [comment]
> omg

"first make it work, then make it better" :smile:

File: src/app/Services/Worksheet/Row.php [hunk 1]
```diff
@@ -10,4 +10,15 @@ public function isRejected()
     {
         return $this->has(config('enso.imports.errorColumn'));
     }
+
+    public function isEmpty()
+    {
+        $validCell = $this->values()->first(function ($cell) {
+            return $cell !== null;
+        });
```
Comment (aocneanu) [inline src/app/Services/Worksheet/Row.php]: refactor to `return $this->values()->filter()->isNotEmpty()`

File: src/app/Services/Worksheet/Row.php [hunk 2]
```diff
@@ -10,4 +10,15 @@ public function isRejected()
...
+
+        return $validCell === null
+            ? true
+            : false;
```
Comment (aocneanu) [inline src/app/Services/Worksheet/Row.php]: equivalent to `return $validCell === null` ...

File: src/app/Services/Importers/Import.php [hunk 3]
```diff
@@ -111,7 +111,10 @@ private function prepareChunk()
         $this->chunk = collect();
 
         while (! $this->sheetHasFinished() && $this->chunkIsIncomplete()) {
-            $this->chunk->push($this->row());
+            $currrentRow = $this->row();
```
Comment (aocneanu) [inline src/app/Services/Importers/Import.php]: rename $currentRow -> $row

File: src/app/Services/Worksheet/Row.php [hunk 4]
```diff
@@ -10,4 +10,15 @@ public function isRejected()
     {
         return $this->has(config('enso.imports.errorColumn'));
     }
+
+    public function isEmpty()
```
Comment (aocneanu) [inline src/app/Services/Worksheet/Row.php]: we need `isNotEmpty()`

File: src/app/Services/Importers/Import.php [hunk 5]
```diff
@@ -111,7 +111,10 @@ private function prepareChunk()
...
+            if (! $currrentRow->isEmpty()) {
```
Comment (aocneanu) [inline src/app/Services/Importers/Import.php]: `$row->isNotEmpty()`

**Review by aocneanu — CHANGES_REQUESTED** [review]

**Pull request merged**